### ISSUE TITLE
Xbox Series X Controller

### ISSUE THREAD

**Issue by lamyipming** [moonlight-stream/moonlight-android#947]
The new Xbox Series X controller is connected to android via LE bluetooth. It works in Parsec and Steam Link. However seems it isn't being recognized in Moonlight Android client. 

Are there any settings I can fiddle with?

**Comment by cgutman** [comment]
It works for me on my Pixel 4 running Android 11.

Please describe your exact hardware and software scenario. Please also confirm that it's not a host-side problem by testing with the on-screen controls.

**Comment by lamyipming** [comment]
Thanks for suggesting the test with on-screen virtual controls. They don't work. Looks like a host problem.

I have tried reinstalling the GeForce Experience. Still, Steam cannot detect my on-screen controller input. I have also tried to disable all the "configuration support" in steam's controller settings. I have also tried to use Steam beta.

I'm on windows 10 with latest update installed.
On client side it is a Samsung tab s7+, Android 11.

**Comment by cgutman** [comment]
Start a stream of your full desktop with Moonlight then open Device Manager, go to View > Devices By Connection, then look for NVVHCI Enumerator. Check that it has some child devices (should be 2 HID devices for keyboard/mouse and your controller). If there are any "Unknown devices" under there, that's your problem.

**Comment by lamyipming** [comment]
The tree in device manager looks like this:
NVVHCI Enumerator
+ USB Input Device
++ HID-compliant mouse
++ HID-compliant mouse
+ USB Input Device
++ HID Keyboard Device

There are no controllers nor unknown devices.

**Comment by cgutman** [comment]
Oh, do you still have on-screen controls enabled? You'll need a controller connected to the client to make the controller show up in Device Manager.

This is what mine looks like (including the virtual Xbox gamepad):
![image](https://user-images.githubusercontent.com/2695644/110219154-2adeeb80-7e83-11eb-9560-d1b03dd0f9ce.png)

**Comment by lamyipming** [comment]
I have the on-screen controls enabled. I have also tried to connect xbox series x controller and a Moga 2 Pocket controller via bluetooth to my android client. But the controller never shows up under NVVHCI emumerator.

I have also tried toggling the Xbox driver setting in the android client, as well as the custom driver option.

Seems like for some reasons my PC never recognize the controllers from moonlight, be it virtual or connected via bluetooth. I'm thinking this is a geforce issue instead of a moonlight issue. 

Sir, I appreciate your continuous effort in helping others in all these years. Thank you for the commitment. As you pointed out, xbox Series X controller is working fine, so I think it's ok to close this thread.
